draft/implementation-notes/index.html

```diff
@@ -538,6 +538,138 @@ <h2>File Purging</h2>
       </p>
     </section>
 
+    <section id="digest-migration">
+      <h2>Migrating to a New Digest Algorithm</h2>
+      <p>
+        Over time new digest algorithms are developed to increase security and address vulnerabilities in existing
+        algorithms. It may become desirable to migrate an object to use a new algorithm while retaining
+        <a href="#version-immutability">Version Immutability</a>. OCFL supports this through the creation of a new
+        version with a new <code>digestAlgorithm</code> that either retains the same object content or is combined with
+        a content update.
+      </p>
+      <p>
+        Consider an example OCFL object where the <code>digestAlgorithm</code> in the <code>inventory.json</code> is
+        <code>sha256</code> and the OCFL object contains a single file (<code>file.txt</code>). We will illustrate
+        migration to use <code>sha512</code> without changing the object content. The starting <code>v1</code> file
+        layout is:
+      </p>
+      <pre>
+[object root]
+    ├── 0=ocfl_object_1.0
+    ├── inventory.json
+    ├── inventory.json.sha256
+    └── v1
+        ├── inventory.json
+        ├── inventory.json.sha256
+        └── content
+            └── file.txt
+      </pre>
+      <p>
+        and the corresponding inventory is:
+      </p>
+      <pre>
+{
+  "digestAlgorithm": "sha256",
+  "head": "v1",
+  "id": "http://example.org/digest_update_example",
+  "manifest": {
+    "579391...bfe": [
+      "v1/content/file.txt"
+    ]
+  },
+  "type": "https://ocfl.io/1.0/spec/#inventory",
+  "versions": {
+    "v1": {
+      "created": "2020-01-01T01:01:01",
+      "message": "sha256 forever",
+      "state": {
+        "579391...bfe": [
+          "file.txt"
+        ]
+      },
+      "user": {
+        "address": "secret@example.org",
```

Comment:
Address should be URI format mailto:secret@example.org?

Reply:
Agreed. Will fix